src/shardeum/shardeumFlags.ts

```diff
@@ -279,7 +280,8 @@ export const ShardeumFlags: ShardeumFlags = {
   unifiedAccountBalanceEnabled: true,
   failedStakeReceipt: true,
   debugDefaultBalance: '100', //In debug mode the default value is 100 SHM.  This is needed for certain load test operations
-  disableSmartContractEndpoints: true // Disable smart contract read endpoints by default
+  disableSmartContractEndpoints: true, // Disable smart contract read endpoints by default
+  stakeLockTime: 1000 * 60 * 60 * 24 * 14, // 2 weeks in milliseconds
```

Comment:
can this use an env variable in addition to the hardcoded default?

Reply:
Done